test/errors_test.rb

```diff
@@ -65,4 +65,23 @@ def test_value_overwrite
     str = "a = false\na = true"
     assert_raises(TOML::ValueOverwriteError) { TOML.parse(str) }
   end
+
+  def test_table_overwrite
```

Comment:
Assignment Branch Condition size for test_table_overwrite is too high. [17.89/15]